### PR TITLE
docs: :pencil: Update READ.ME

### PR DESCRIPTION
**Pull request by RayeEThompson**
Updated READ.ME to have additional developer support: 
- abstract for branch
- channels documentation

### CODE REVIEW

**Comment by RayeEThompson** [comment]
Hey there @becomevocal, here is the PR with the updates we spoke of. We will be putting the documentation to support this effort in the developer library.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# Gatsby + BigCommerce + Netlify CMS Starter
+# Gatsby + BigCommerce + Netlify CMS Starter: Mutli-Region
```
Comment (becomevocal) [inline README.md:1]: "Multi" instead of "Mutli"?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
-# Gatsby + BigCommerce + Netlify CMS Starter
+# Gatsby + BigCommerce + Netlify CMS Starter: Mutli-Region 
 
-Note: This starter uses Gatsby v2.
+## Abstract
+
+The following is an overview of the creation of a headless multi-region eCommerce site using BigCommerce APIs. There was a presentation given at the Netlify Headless Commerce Summit on September 3, 2020. You will find references and materials to support the development of a multi-region headless site using GatsbyJS, BigCommerce, and Netlify in the [BigCommerce Developer Library](https://developer.bigcommerce.com/dev-library/).
```
Comment (becomevocal) [inline README.md]: Once the entry is up, we should link directly to it.

**Comment by RayeEThompson** [comment]
@becomevocal Dev Library is deployed and the links are updated.

**Review by becomevocal — APPROVED** [review]
Awesome 👍  💯

**Pull request merged**